src/routes/docs/tooling/command-line/functions/+page.markdoc

```diff
@@ -0,0 +1,172 @@
+---
+layout: article
+title: Deploy functions with CLI 2.0
+description: Efficiently deploy your Appwrite functions using the Command-Line Tool (CLI).
+---
+
+The CLI handles the creation and deployment of Appwrite Functions. Install and initialize CLI Appwrite [here](/docs/tooling/command-line/installation#Installation).
+
+# Pull function {% #pull-function %}
+
+You can pull your Appwrite functions customizable settings using the `pull` command in the folder holding your `appwrite.json` file.
+
+```sh
+appwrite pull functions
+```
+
+# Appwrite.json {% #appwritejson %}
```

Comment:
Since this is a file name, we can have the title all lowercase otherwise it can be misleading and cause devs to spell the filename different than what expected.